cfy_cluster_setup/main.py

```diff
@@ -494,6 +562,12 @@ def main():
         default=False,
         help='Using a three nodes cluster.')
 
+    generate_config_args.add_argument(
+        '--external-db',
+        action='store_true',
```

Comment:
Should this maybe want an address (IP or hostname) instead of just being a flag?

Reply:
It wants a lot more than that, unfortunately. This is why I added a [whole section for this use case in the configuration file](https://github.com/cloudify-cosmo/cloudify-cluster-setup/blob/e55cadd89cf2cb4c5d24f022e1c442dcfdb31a9e/cfy_cluster_setup/cfy_cluster_config_files/cfy_external_db_cluster_config.yaml#L65-L86).

Reply:
Oh, that sucks. Fair enough then, let's go with this route for now and see if we can do anything better with some background cfy_manager improvements in future.

Reply:
Sounds good.